packages/dcos-integration-test/extra/conftest.py

```diff
@@ -21,6 +21,31 @@
 LOGIN_PW = os.getenv('DCOS_LOGIN_PW')
 
 
+def pytest_configure(config):
+    config.addinivalue_line(
+        'markers', 'order(n): mark test to run in Nth order.'
+        'Negative numbers represent position from the end.'
+        'EG: 1, 2, 3, ... unordered ... -3, -2, -1')
+
+
+def pytest_collection_modifyitems(session, config, items):
+    """Reorders test using order mark
+    """
+    first = []
+    unordered = []
+    last = []
+    for item in items:
+        if not hasattr(item.obj, 'order'):
```

Comment:
We can do [1, 2, 3, 0, -3, -2, -1] if you want 1 to always be first. They just need to be opposite orderings, if I put an initalizer in 2, it should run before 3, and if I put a finalizer in -2 for that same thing, it should come after -3. The "three" level always comes before or always comes after.

```
assert all(lambda x: x < 1000, items)
sorted([-2, 1, 3, -1, 2, 0, -3], key= lambda x: 1000-x)
```
gives the `[3, 2, 1, 0, -1, -2, 3]` ordering